GVFS/GVFS.Service/GvfsService.cs

```diff
@@ -61,6 +60,13 @@ public void Run()
 
                     this.serviceStopped.WaitOne();
                 }
+
+                // https://github.com/Microsoft/VFSForGit/issues/774.
+                // Start product upgrade timer only after attempting to enable prjflt.
+                // On Windows server (where PrjFlt is not inboxed) this helps avoid
+                // a race between TryEnablePrjFlt() and installer pre-check which is
+                // performed by UpgradeTimer in parallel.
+                this.productUpgradeTimer.Start();

```

Comment:
Do we need to check the error from the `TryEnablePrjFlt`?  Doesn't look like we were doing anything with it before which is suspicious.

Reply:
This is by design. Failure in `TryEnablePrjFlt` should not stop service startup. Service makes the best-effort, logs failure and proceeds. When user mounts an enlistment, `gvfs mount` will again try to enable `PrjFlt`. 

Added a comment to document this behavior. Also corrected the error messaging in `gvfs mount` when enable `PrjFlt` fails.